ms-windows/templates/3.0/x64/openssl.iss

```diff
@@ -0,0 +1,152 @@
+; Script generated by the Inno Setup Script Wizard.
+; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
+
+[Setup]
+AppName=OpenSSL (64-bit)
+AppVersion=[[DOTTED_VERSION]]
+AppVerName=OpenSSL [[DOTTED_VERSION]] (64-bit)
+AppPublisher=OpenSSL Installer Team
+AppPublisherURL=https://www.openssl.org
+AppSupportURL=https://github.com/openssl/installer
+AppUpdatesURL=https://www.openssl.org
+VersionInfoVersion=[[WINDOWS_DOTTED_VERSION]]
+DefaultDirName={pf}\OpenSSL
+DisableDirPage=no
+DefaultGroupName=OpenSSL
+SourceDir=[[INSTALLER_SOURCE_DIR]]
+OutputBaseFilename=OpenSSL-[[DOTTED_VERSION]]-x64
+LicenseFile=license.txt
+; SetupIconFile=compiler:SetupClassicIcon.ico

```

Comment:
Remove?

Reply:
Actually, this might be necessary if XP support is dropped from the installers.  I don't think the new default Inno Setup 6 installer icon looks all that great.  The old icon is blocky/chunky but the new icon doesn't even look like an installer.

We could also supply our own icon for the installer package instead of the generic Inno Setup icon.  Making a Windows .ico file that works at all possible resolutions is a bit of an arcane artform.